Extensions/TileMap/JsExtension.js

```diff
@@ -832,7 +847,7 @@ const defineSimpleTileMap = function (extension, _, gd) {
       _('Flip tile vertically (at position)'),
       _('Flip tile vertically at scene coordinates.'),
       _(
-        'Flip tile vertically at scene coordinates _PARAM1_ ; _PARAM2_: _PARAM3_'
+        'Flip tile vertically at scene coordinates _PARAM1_ ; _PARAM2_: _PARAM3_ of _PARAM0_'
```

Comment:
Everywhere: don't put "of <object name>" after a boolean but after the scene coordinates.

For coordinates, use `X;Y`, not `X ; Y`